src/back/glsl/mod.rs

```diff
@@ -2006,10 +2014,11 @@ impl<'a, W: Write> Writer<'a, W> {
                                         if let Some(crate::Binding::BuiltIn(builtin)) =
                                             member.binding
                                         {
+                                            has_point_size |= builtin == crate::BuiltIn::PointSize;
+
                                             match builtin {
                                                 crate::BuiltIn::ClipDistance
-                                                | crate::BuiltIn::CullDistance
-                                                | crate::BuiltIn::PointSize => {
```

Comment:
@teoxoy I removed `BuiltIn::PointSize` from the list to let it write that built-in. But I'm not sure it is going to work. Could you provide some context or example of input that will trigger this code to execute? I can then debug and get better understanding.

Reply:
I think it should be fine since `gl_PointSize` is available in gles (see https://docs.gl/), the others are not.

Reply:
It might be worth turning this into an error instead of silently ignoring them, but that could be another PR.

Reply:
@teoxoy ClipDistance and CullDistance are mentioned in https://registry.khronos.org/OpenGL/specs/es/3.2/GLSL_ES_Specification_3.20.htm in 14.2.5. Mapping Variables. What is the reason it is skipped in case of ES in the code?

![image](https://user-images.githubusercontent.com/5107562/215767874-64cc6a4c-0f97-4722-913e-292af7fb6c46.png)

Reply:
Ah, I see. We need to check for a gles version >= 3.2 then. I think the check was incomplete since WebGL is based on gles 3.0 and most people using the backend target WebGL.

Reply:
@teoxoy just found this https://github.com/KhronosGroup/GLSL/issues/132, so added warning log in case of use `ClipDistance` and `CullDistance` in unsupported version

Reply:
I see, so the GLSL ES spec is wrong here and no gles version supports them.